src/loader.rs

```diff
@@ -48,6 +48,30 @@ pub struct LoaderMetadata {
     encrypted_bytes: CipherText,
 }
 
+// Loader for using in tests.  Just creates defualt metadata and Keytree.
+// This shouldn't be used for production, only tests.
+// type Meta = () would be more appropriate for this but it changes the
```

Comment:
Maybe `struct TrivialKeystoreLoader<Meta>` and `impl<L: Ledger, Meta> KeystoreLoader<L> for TrivialKeystoreLoader<Meta>`?

Reply:
I moved this to `mods.rs` and it's not used anywhere else now including cli so that should make the comment in the code irrelevant now.